libcomps/src/python/tests/comps/comps_empty_by_lang_tags.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <group>
+   <id>birds</id>
+   <default>true</default>
+   <uservisible>true</uservisible>
+   <display_order>1024</display_order>
+   <name>birds</name>
+   <description></description>
+    <packagelist>
+      <packagereq type="mandatory">cockateel</packagereq>
+      <packagereq type="mandatory">duck</packagereq>
+      <packagereq type="mandatory">penguin</packagereq>
+      <packagereq type="mandatory">stork</packagereq>
+    </packagelist>
+  </group>
+  <group>
+   <id>mammals</id>
+   <default>true</default>
+   <uservisible>true</uservisible>
+   <display_order>1024</display_order>
+   <name>mammals</name>
+   <description></description>
+    <packagelist>
+      <packagereq type="mandatory">bear</packagereq>
+      <packagereq type="mandatory">camel</packagereq>
+      <packagereq type="mandatory">cat</packagereq>
+      <packagereq type="mandatory">cheetah</packagereq>
+      <packagereq type="mandatory">chimpanzee</packagereq>
+      <packagereq type="mandatory">cow</packagereq>
+      <packagereq type="mandatory">dog</packagereq>
+      <packagereq type="mandatory">dolphin</packagereq>
+      <packagereq type="mandatory">elephant</packagereq>
+      <packagereq type="mandatory">fox</packagereq>
+      <packagereq type="mandatory">giraffe</packagereq>
+      <packagereq type="mandatory">gorilla</packagereq>
+      <packagereq type="mandatory">horse</packagereq>
+      <packagereq type="mandatory">kangaroo</packagereq>
+      <packagereq type="mandatory">lion</packagereq>
+      <packagereq type="mandatory">mouse</packagereq>
+      <packagereq type="mandatory">squirrel</packagereq>
+      <packagereq type="mandatory">tiger</packagereq>
+      <packagereq type="mandatory">walrus</packagereq>
+      <packagereq type="mandatory">whale</packagereq>
+      <packagereq type="mandatory">wolf</packagereq>
+      <packagereq type="mandatory">zebra</packagereq>
+    </packagelist>
+  </group>
+  <category>
+   <id>all</id>
+   <display_order>99</display_order>
+   <name>all</name>
+   <description></description>
+    <grouplist>
+     <groupid>mammals</groupid>
+     <groupid>birds</groupid>
+    </grouplist>
+  </category>
+  <langpacks>
+    <match install="gorilla-%s" name="gorilla-en"/>
+    <match install="zebra-%s" name="zebra-docs"/>
+  </langpacks>
+</comps>
```

Comment:
This looks like it contains data irrelevant to the test, could it be minimized to only contain what is important for the test?

Reply:
I remove all that was unnecessary while keeping the combinations of empty/missing tags.

Reply:
Much better, cheers :slightly_smiling_face:

(Also, I wouldn't mind having the test in the same commit with the fix, but that doesn't matter much... merging.)